aten/src/ATen/native/ts_native_functions.yaml

```diff
@@ -142,40 +147,59 @@ full_codegen:
   - upsample_nearest2d_backward
   - zero.functional
   - narrow_copy.SymInt
+  - alias_copy
+  - as_strided_copy
+  - diagonal_copy
+  - expand_copy
+  - permute_copy
+  - _reshape_alias_copy
+  - select_copy.int
+  - detach_copy
+  - slice_copy.Tensor
+  #- split_copy.Tensor
+  #- split_with_sizes_copy
+  #- unbind_copy.int
+  - squeeze_copy
+  - squeeze_copy.dim
+  - t_copy
+  - transpose_copy.int
+  - unsqueeze_copy
+  - view_copy
+  - view_copy.dtype
+  - unfold_copy
+  - select_scatter
+  - slice_scatter
+  - diagonal_scatter
+  - as_strided_scatter
 supported:
-  - as_strided
-  - as_strided_
-  - clone
   - _copy_from
   - _copy_from_and_resize
-  - diagonal
   - empty.memory_format
   - empty_strided
-  - expand
   - fill_.Scalar
-  - narrow
   - native_batch_norm
   - native_batch_norm_backward
   - normal_
   - max_pool3d_with_indices
   - max_pool3d_with_indices_backward
-  - permute
-  - select.int
-  - slice.Tensor
-  - squeeze
-  - squeeze.dim
-  - squeeze_
-  - squeeze_.dim
-  - t
-  - t_
   - _to_copy
-  - transpose.int
-  - transpose_
-  - unsqueeze
-  - unsqueeze_
-  - view
```

Comment:
further down, there are still view ops in the 'non_native' section, which means some of their impl gets codegenned still.  You might have left this intentionally (to minimize the size of this PR and clean up later) - if so, disregard.  Guessing this won't hurt anything.

Reply:
yep exactly (I could chosen to keep all of the view ops in and clean them up later, but removing the codegen'd view ops was helpful to surface errors. Removing them made it an error for accidentally send a view op to LTC).

Removing the hand-written ones require ripping out more infra, so I left that for a later PR